BD-1372: Adding integration specifications for CC Web SDK (#3619)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/web/content_cards/integration.md

# Content Card integration

{% include archive/web-v4-rename.md %}

## Content Card data model {#data-models}

The Content Cards data model is available in the Web SDK.

## Content Card model
The Braze Web SDK offers three Content Card types: [Banner](https://js.appboycdn.com/web-sdk/latest/doc/classes/braze.banner.html), [CaptionedImage](https://js.appboycdn.com/web-sdk/latest/doc/classes/braze.captionedimage.html), and [ClassicCard](https://js.appboycdn.com/web-sdk/latest/doc/classes/braze.classiccard.html). Each type inherits common properties from a base model [Card](https://js.appboycdn.com/web-sdk/latest/doc/classes/braze.card.html) and has the following additional properties.

### Base Content Card model properties - Card

|Property|Description|
|---|---|
| `expiresAt` | The unix timestamp of the card's expiration time.|

> | `extras`| (Optional) An object of string/string key/value pairs.|

Reviewer: Is this correct? The slash is a bit confusing here. 🤔 
```suggestion
| `extras`| (Optional) An object of string or string key-value pairs.|
```

> | `extras`| (Optional) An object of string/string key/value pairs.|

Author: ```suggestion
| `extras`| (Optional) Key-value pair data formatted as a string object with a value string. |
```

}
```

{% alert note %}

> Visit the following customization articles for documention on adding [custom UI]({{site.baseurl}}/developer_guide/platform_integration_guides/web/content_cards/customization/custom_ui/), [custom styling]({{site.baseurl}}/developer_guide/platform_integration_guides/web/content_cards/customization/custom_styling), [key-value pairs]({{site.baseurl}}/developer_guide/platform_integration_guides/web/content_cards/customization/key_value_pairs), [read and unread indicators]({{site.baseurl}}/developer_guide/platform_integration_guides/web/content_cards/customization/read_and_unread/), and [requesting unviewed Content Card counts]({{site.baseurl}}/developer_guide/platform_integration_guides/web/content_cards/customization/badges).

Author: ```suggestion
Visit the following customization articles for documentation on adding [custom UI]({{site.baseurl}}/developer_guide/platform_integration_guides/web/content_cards/customization/custom_ui/), [custom styling]({{site.baseurl}}/developer_guide/platform_integration_guides/web/content_cards/customization/custom_styling), [key-value pairs]({{site.baseurl}}/developer_guide/platform_integration_guides/web/content_cards/customization/key_value_pairs), [read and unread indicators]({{site.baseurl}}/developer_guide/platform_integration_guides/web/content_cards/customization/read_and_unread/), and [requesting unviewed Content Card counts]({{site.baseurl}}/developer_guide/platform_integration_guides/web/content_cards/customization/badges).
```
